db/migrate/20170513172145_add_medical_review_to_dogs.rb

```diff
@@ -0,0 +1,5 @@
+class AddMedicalReviewToDogs < ActiveRecord::Migration[5.0]
+  def change
+    add_column :dogs, :medical_review_complete, :boolean, default: false
```

Comment:
I think the cap deploy is setup to handle the db updates before the code goes out, so I think we're good.  Thanks!